lib/teleterm/apiserver/handler/handler_access_requests.go

```diff
@@ -48,22 +48,20 @@ func (s *Handler) GetAccessRequests(ctx context.Context, req *api.GetAccessReque
 	return response, nil
 }
 
-// GetAccessRequest returns a single access request by id
+// GetAccessRequest returns a single access request by id.
 func (s *Handler) GetAccessRequest(ctx context.Context, req *api.GetAccessRequestsRequest) (*api.GetAccessRequestResponse, error) {
 	requests, err := s.DaemonService.GetAccessRequests(ctx, req)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
 	response := &api.GetAccessRequestResponse{}
-	if len(requests) < 1 {
-		return nil, trace.NotFound("access request %q not found", req.Id)
-	}
```

Comment:
Oh, but this was a good check, wasn't it? Now the GetAccessRequest will return a successful response if the request wasn't found. I suppose we could still do this check within `daemon.Service.GetAccessRequest`?

Reply:
I moved this into `daemon.Service.GetAccessRquest` in the previous push.

Reply:
> I moved this into `daemon.Service.GetAccessRquest` in the previous push.

I don't understand, I don't see this logic being moved to `daemon.Service.GetAccessRquest`. 🤔

Reply:
Weird, I also don't see it when clicking on the file name from this comment. I don't see the `GetAccessRequest` at all. But I see it in the diff [here](https://github.com/gravitational/teleport/pull/16694/files#diff-c990813ef25502ef5bc40aff2882ccd6c61edf5c4d807d78780ced8e7d923d6eR388)